### ISSUE TITLE
PM2 broken after complete deletion of this plugin

### ISSUE THREAD

**Issue by valentincognito** [jon-hall/pm2-windows-service#33]
I tried to work with this plugin that looked pretty good but it didn't work for me.. I uninstalled the service and the the package. But now my regular install of pm2 doesn't work anymore... Seems like pm2 is running in a wrong directory..

I tried to uninstall pm2 and reinstall it again but it didn't work.

Any bit of help very much appreciated

**Comment by jon-hall** [comment]
Hi @valentincognito, I'm sorry this package wasn't what you needed and hopefully I'll be able to figure out why you're now having issues with pm2 itself.

The first thing to verify is that the service has definitely been uninstalled - check your Services list in Windows and confirm it's no longer installed or running.  If the service is gone, then my guess as to why pm2 is no longer working for you is because the environment setup done by the service doesn't currently rollback when you uninstall it.  Check your machine-level environment variables to see if there's a value of `PM2_HOME` set which could be continuing to impact where pm2 runs from, if so then clear it or point it to the directory you want it to run from.

Let me know if that's still not working for you.

**Comment by valentincognito** [comment]
Hi @jon-hall, I appreciate the answer ! I already tried all your suggestions but it didn't fix it. It's a tricky problem because I can start a nodejs server and it works. I just noticed before that only my servers that uses the fs module causes problems because fs seems lost and scan wrong directories. (those same servers work well with forever or a npm start, they only crash with pm2)

I was also certain the PM2_HOME value would fix the whole thing but it didn't... 

I am not even sure if I can blame your package anymore, maybe while cleaning the services and the environment variables I did something wrong... 

If you have no further idea where to look I don't want to bother you too much, I'll rollback my machine , close this issue and call it a day ^^

**Comment by jon-hall** [comment]
This may still be fixable, there's potentially a few other things which could be causing the unexpected behaviour.

When you run `pm2 status` on the commandline what comes up? Does `pm2 status` work from a non-admin prompt?

Is the application started using a simple `pm2 start index.js` or is it using a JSON config file?  If it's using a config file, are you able to post up a (sanitised) version?

Are you able to get the application to log or dump what it's CWD is, and if so, what is the CWD?  And what would you expect the CWD to be?

Those are the guesses that are top of my list after confirming PM2_HOME wasn't interfering, hopefully there might be something revealing what's going on in there.

**Comment by valentincognito** [comment]
The pm2 status showed my applications running, and yest I start them with pm2 start index.js (no JSON file) and the working directory was correct. 

I am not sure but I think I found what was the problem and it was indeed not related to your package anymore. I reinstalled gitbash and it seems to work. Maybe some permissions got lost along the way :(

I cannot say if it was working or not with the windows cmd at the time of the bug because I didn't try it.

Anyway sorry for the troubles and thanks for the help !
